rpc/src/v1/helpers/signing_queue.rs

```diff
@@ -100,86 +99,77 @@ pub trait SigningQueue: Send + Sync {
 
 #[derive(Debug, Clone, PartialEq)]
 /// Result of a pending confirmation request.
-pub enum ConfirmationResult {
-	/// The request has not yet been confirmed nor rejected.
-	Waiting,
+// TODO [kirushik] Let's get rid of this completely, and simplify that `Result<Option<Result...` juggling somehow
```

Comment:
Let's do it in this PR then, a typedef might help.

Reply:
Fair point